documentation/blog.md

```diff
@@ -0,0 +1,27 @@
+As you already know, YaST was converted from YCP to Ruby some time ago. At least we repeat it quite often ;-) However, this conversion was done on language basis. Some old design decissions and principles stayed. One of them is usage of SCR for accessing underlying configuration files.
+
+SCR was designed together with YaST. It uses concept of "agents" for accessing configuration files. These agents contains a description of configuration file using own format. Moreover SCR offers location transparency. You can e.g. work with a file in system or in a chrooted environment. However, this piece of code is proprietaty and limited by different quality of agents. It is also written in C++ and development is done only in SUSE. From this and some other reasons we decided to replace proprietary SCR with something else. We started to use "Config files API"
```

Comment:
maybe also add in this paragraph, that SCR have limited reusability. you cannot use just parser or just location transparency easily. Or switch to use different parser.